src/Blueprint/Blueprint/Cmdlets/RemoveAzureRMBlueprintAssignment.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Azure.Commands.Blueprint.Models;
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.PowerShell.Cmdlets.Blueprint.Properties;
+using ParameterSetNames = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterSetNames;
+using ParameterHelpMessages = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterHelpMessages;
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintAssignment", SupportsShouldProcess = true)]
+    public class RemoveAzureRmBlueprintAssignment : BlueprintCmdletBase
+    {
+        #region Parameters
+        [Parameter(ParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByName, Position = 0, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.SubscriptionId)]
+        [ValidateNotNullOrEmpty]
+        public string SubscriptionId { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByName, Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.BlueprintAssignmentName)]
+        [ValidateNotNull]
+        public string Name { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByObject, Position = 0, Mandatory = true, ValueFromPipeline = true, HelpMessage = ParameterHelpMessages.BlueprintAssignmentObject)]
+        public PSBlueprintAssignment BlueprintAssignmentObject { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
```

Comment:
>PassThru [](start = 31, length = 8)

What does that do?

Reply:
This is to provide output to the user when no output would normally be provided. It's a requirement for Remove-* cmdlets. [https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/patterns-for-standard-cmdlets.md#remove--cmdlet](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/patterns-for-standard-cmdlets.md#remove--cmdlet)

---
In reply to: [257385808](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257385808) [](ancestors = 257385808)